ui/src/app/workflows/components/workflow-dag/workflow-dag.scss

```diff
@@ -2,143 +2,66 @@
 @import 'node_modules/@fortawesome/fontawesome-free/scss/variables';
 @import 'node_modules/@fortawesome/fontawesome-free/scss/mixins';
 
-.workflow-dag {
-    position: relative;
-
-    &__line {
-        position: absolute;
-        border-top: 1px solid $argo-color-gray-5;
-    }
-
-    &__title {
-        font-size: 50%;
-    }
+.workflow-dag-render-options-panel {
+  position: fixed;
+  margin: 10px;
+  padding: 10px;
+  display: inline-block;
+  background-color: white;
+  box-shadow: 1px 1px 3px $argo-color-gray-4;
+
+  a {
+    padding: 10px;
+    color: $argo-color-gray-6;
+  }
+}
 
-    &__edge {
-        .workflow-dag__line {
-            &:last-child:not(.workflow-dag__line--no-arrow) {
-                &:after {
-                        content: '\25BA';
-                        position: absolute;
-                        color: #A3A3A3;
-                        font-size: 12px;
-                        top: -9px;
-                        left: -1px;
-                        transform: rotate(180deg);
-                    }
-                }
-            }
-        }
+.workflow-dag {
 
-    &__node {
-        position: absolute;
-        box-shadow: 2px 2px 2px $argo-color-gray-4;
-        cursor: pointer;
+  text-align: center;
 
-        &.active {
-            border-color: $argo-color-teal-6;
-        }
+  .line {
+    fill: transparent;
+    stroke-width: 2px;
+    stroke: $argo-color-gray-4;
+  }
 
-        &.small {
-            background-color: transparent;
-            border: none;
-            padding-left: 0;
+  .label {
+    stroke: $argo-color-gray-7;
+    font-weight: lighter;
+  }
```

Comment:
This stroke and font weight makes the font hard to read in my opinion. I would suggest removing it.

From
<img width="101" alt="image" src="https://user-images.githubusercontent.com/614838/81215289-fb730000-8f8d-11ea-8015-7cbcf3b74b37.png">

To:
<img width="116" alt="image" src="https://user-images.githubusercontent.com/614838/81215321-088fef00-8f8e-11ea-9a9c-e4ac9afed53a.png">

Reply:
Ok. So I've corrected the font used and increased the font size to 12pt.